src/olympia/migrations/952-correct-index-theme-user-counts.sql

```diff
@@ -0,0 +1,5 @@
+-- Drop old index first
+DROP INDEX `addon_date_idx` ON `theme_user_counts`;
+
+-- Now create a proper
+ALTER TABLE `theme_user_counts` ADD CONSTRAINT `theme_user_counts_date_cc9034dde90789f_uniq` UNIQUE (`date`, `addon_id`);
```

Comment:
If I drop my table locally and recreate it from `sqlall` I don't get that. I'm not sure if `sqlall` is reliable these days, though...